### PR TITLE
fix: Fix for camera reset position

### PR DESCRIPTION
**Pull request by samjosephgates**
### Testing 🧪
 > [ Add any special instructions needed to test or view your changes such as environment URLs or other config details. ]

### Checklist ✔️
- [ ] Linked associated issue (if present)
- [ ] Added relevant labels
- [ ] Requested developer reviews
- [ ] Request UI review from design / PM
- [ ] Verify all code checks are passing

### CODE REVIEW

File: src/Components/3DV/SceneView.tsx [hunk 1]
```diff
@@ -344,7 +346,7 @@ function SceneView(props: ISceneViewProps, ref) {
                     const es = someMeshFromTheArrayOfMeshes.getBoundingInfo()
                         .boundingBox.extendSize;
                     const es_scaled = es.scale(
-                        zoomMeshIds && zoomMeshIds.length < 10 ? 5 : 3
+                        zoomMeshIds && zoomMeshIds.length > 10 ? 5 : 3
```
Comment (msnyder-msft) [inline src/Components/3DV/SceneView.tsx:349]: why is 10 the magic number here?

File: src/Components/3DV/SceneView.tsx [hunk 2]
```diff
@@ -379,6 +381,10 @@ function SceneView(props: ISceneViewProps, ref) {
                         cameraRef.current.wheelPrecision =
                             (3 * 40) / bbox.boundingSphere.radius;
 
+                        initialCameraTargetRef.current = deepCopy(
```
Comment (msnyder-msft) [inline src/Components/3DV/SceneView.tsx:384]: please add comments. :)

File: src/Components/3DV/SceneView.tsx [hunk 3]
```diff
@@ -592,33 +603,22 @@ function SceneView(props: ISceneViewProps, ref) {
...
+            ? initialCameraTargetRef.current
+            : cameraRef.current.target;
         const radiusTo = radius;
         // Reset camera back to original position
-        cameraRef.current.position = positionFrom;
         cameraRef.current.target = targetFrom;
         const ease = new BABYLON.CubicEase();
         ease.setEasingMode(BABYLON.EasingFunction.EASINGMODE_EASEINOUT);
         if (!zoomOnly) {
-            BABYLON.Animation.CreateAndStartAnimation(
```
Comment (msnyder-msft) [inline src/Components/3DV/SceneView.tsx:611]: why remove only this one? I'm not following why we have multiple anyways, what are they each for? Can we add comments for the rest of us?

Comment (samjosephgates) [reply src/Components/3DV/SceneView.tsx:611]: We had three animations for moving the camera, radius, target and position. In testing, I have realised the position animation is not required.

File: src/Components/3DV/SceneView.tsx (hunk 2 above)
Comment (samjosephgates) [reply src/Components/3DV/SceneView.tsx:384]: Done

File: src/Components/3DV/SceneView.tsx (hunk 1 above)
Comment (samjosephgates) [reply src/Components/3DV/SceneView.tsx:349]: Updated code and added a comment. What we actually want to do here is set the scale to be a little further out if we are zooming into an element vs the whole model.

**Review by msnyder-msft — APPROVED** [review]
Code LGTM

**Comment by msnyder-msft** [comment]
Do we still need this PR? Is it just needing a second review?

**Comment by samjosephgates** [comment]
> Do we still need this PR? Is it just needing a second review?

Yes still required.

**Review by ccrowley96 — APPROVED** [review]

**Pull request merged**